go.mod

```diff
@@ -0,0 +1,29 @@
+module gopkg.in/macaroon-bakery.v2
+
+require (
+	github.com/golang/protobuf v1.0.0
+	github.com/juju/clock v0.0.0-20180524022203-d293bb356ca4 // indirect
+	github.com/juju/errors v0.0.0-20170703010042-c7d06af17c68 // indirect
+	github.com/juju/loggo v0.0.0-20170605014607-8232ab8918d9
+	github.com/juju/postgrestest v1.0.1
+	github.com/juju/retry v0.0.0-20180821225755-9058e192b216 // indirect
+	github.com/juju/schema v0.0.0-20160420044203-075de04f9b7d // indirect
+	github.com/juju/testing v0.0.0-20180820040200-b0b89ba330f2
+	github.com/juju/utils v0.0.0-20180207021810-d18e608d0140 // indirect
+	github.com/juju/version v0.0.0-20161031051906-1f41e27e54f2 // indirect
+	github.com/juju/webbrowser v0.0.0-20160309143629-54b8c57083b4
+	github.com/julienschmidt/httprouter v0.0.0-20151013225520-77a895ad01eb
+	github.com/rogpeppe/fastuuid v0.0.0-20150106093220-6724a57986af
+	golang.org/x/crypto v0.0.0-20180228161326-91a49db82a88
+	golang.org/x/net v0.0.0-20180306060152-d25186b37f34
+	golang.org/x/sync v0.0.0-20180314180146-1d60e4601c6f // indirect
+	golang.org/x/sys v0.0.0-20180302081741-dd2ff4accc09 // indirect
+	gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127
+	gopkg.in/errgo.v1 v1.0.0
+	gopkg.in/httprequest.v1 v1.1.1
+	gopkg.in/juju/environschema.v1 v1.0.0-20151104115810-7359fc7857ab
+	gopkg.in/macaroon.v2 v2.0.0
+	gopkg.in/mgo.v2 v2.0.0-20160818020120-3f83fa500528
+	gopkg.in/yaml.v2 v2.1.1
+	launchpad.net/gocheck v0.0.0-20140225173054-000000000087 // indirect
```

Comment:
I suspect If you update the mgo dependency then this one will go away.

Reply:
done (after a bit of prereq faff). it wasn't the mgo dep.